zzz_generated/app-common/app-common/zap-generated/attributes/Accessors.h

```diff
@@ -41,22 +42,27 @@ namespace Attributes {
 namespace IdentifyTime {
 Protocols::InteractionModel::Status Get(chip::EndpointId endpoint, uint16_t * value); // int16u
 Protocols::InteractionModel::Status Set(chip::EndpointId endpoint, uint16_t value);
+Protocols::InteractionModel::Status Set(chip::EndpointId endpoint, uint16_t value, MarkAttributeDirty markDirty);
```

Comment:
How is the underlying setter supposed to do that?

The only time a MarkAttributeDirty value needs to be passed in here is for "Q" ("quiet") attributes.  And for those attributes sometimes you want to mark dirty and sometimes not, depending on what the spec defines for that specific attribute.

One we actually have a concept of Q in the XML, we can stop generating these extra functions for all the non-Q attributes.  Alternately, we could not generate them for now and only start generating once we have Q in XML....  But in practice, we might want to have the right behavior for Q attributes before we've added Q to XML.

Reply:
It's important for more than just Q. For example, LevelControl reports too frequently, specifically because it uses ::Set() but it calls it too frequently (every 100ms!). I guess it's better to call it with "MarkDirty == false" and do the reporting on its own. Perhaps this is an improvement of sorts, but it's unfortunate it adds 1 arg to every single generated call, even though most of them will say "true"?

Reply:
>  I guess it's better to call it with "MarkDirty == false" and do the reporting on its own.

Right, that would make that a Q attribute (value can change without reporting)....  which it should be in the spec.  ;)